win32ss/user/ntuser/object.c

```diff
@@ -207,12 +207,45 @@ static PVOID AllocSysObject(
     return Object;
 }
 
+_Success_(return!=NULL)
+static PVOID AllocSysObjectCB(
+    _In_ PDESKTOP pDesk,
+    _In_ PTHREADINFO pti,
+    _In_ SIZE_T Size,
+    _Out_ PVOID* ObjectOwner)
+{
+    PVOID Object;
+
+    UNREFERENCED_PARAMETER(pDesk);
+    UNREFERENCED_PARAMETER(pti);
+    ASSERT(Size > sizeof(HEAD));
+
+    /* Allocate the clipboard data */
+    Object =  ExAllocatePoolWithTag(PagedPool, Size, USERTAG_CLIPBOARD);
+
+    if (!Object)
+    {
+        ERR("Object is NULL-No Object Created.\n");
```

Comment:
```suggestion
        ERR("ExAllocatePoolWithTag failed, no object created.\n");
```
I've had hard time trying to understand this text :sweat_smile:

Reply:
Improved. Thanks.